WTWSMS/events/religious_events.txt

```diff
@@ -30,7 +30,15 @@ province_event = {
 	}
 	
 	mean_time_to_happen = {
-		months = 9600
+		months = 8000
+		modifier = {
+			factor = 0.15
+			AND = {
+				NOT = { year = 500 }
+				religion = nicene # In the early parts of this Mod, the religious world was in turmoil with the fall of Christian WRE
```

Comment:
But there is already. Nicene takes a huge penalty leading to religious chaos. What is the difference from the existant mechanic and this one? That yours cannot be averted and that it is ruled by fix dates rather than dynamic. This is railroading at its earnest, and conflicts directly with the way the Fall of Rome is portrayed.